### PR TITLE
Adding translations for ca-ES and es-ES.

### PR DESCRIPTION
**Pull request by iwalkalone69**

### CODE REVIEW

**Comment by wenzhixin** [comment]
In fact, you don't need to add the dist file.

**Comment by iwalkalone69** [comment]
Sorry, I didn't know about the dist file. es-ES fixed also.

File: src/locale/bootstrap-table-ca-ES.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+/**
+ * Bootstrap Table English translation
+ * Author: Zhixin Wen<wenzhixin2010@gmail.com>
```
Comment (wenzhixin) [inline src/locale/bootstrap-table-ca-ES.js]: Please update your name here:)

Comment (iwalkalone69) [inline src/locale/bootstrap-table-ca-ES.js]: Done. Thanks.

**Pull request merged**